caddyhttp/browse/browse_test.go

```diff
@@ -300,7 +300,7 @@ func TestBrowseJson(t *testing.T) {
 		req.Header.Set("Accept", "application/json")
 		rec := httptest.NewRecorder()
 
-		code, err := b.ServeHTTP(rec, req)
+		code, _ := b.ServeHTTP(rec, req)
```

Comment:
Ahhh, I figured it out -- it's not your fault, @Echsecutor. ineffassign [was updated just a few days ago](https://github.com/gordonklaus/ineffassign/commit/99762ed65232b1477364c8c53778f8b6503e868c) to expose more ineffectual assignments. My local clone needed to be updated and now I see the same warnings. And it just so happens that this is the first pull request since that commit so it's the first one to show the warnings.

Thanks for addressing them!